src/Core/tests/DeviceTests/Stubs/EllipseStub.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui.DeviceTests.Stubs
+{
+	public class EllipseStub : StubBase, IShape
+    {
+		public PathF PathForBounds(Rectangle rect)
+		{
+			var path = new PathF();
+
+			path.AppendEllipse(0f, 0f, (float)Width, (float)Height);
+
+			return path.AsScaledPath((float)Width / (float)rect.Width);
```

Comment:
Looking at this reminds me of the old code. Maybe these methods should be helper methods in Core? Like PathHelper.CreateEllipse and pass in the relevant props. This way we can test the logic in the actual implementation. Or, do we even need all the shapes in the tests since they are not serving any purpose? We are not testing the logic in Controls, so why even have this shape?

I should merge my device tests for compat/controls PR...